src/table/pagination-table/components/styles.ts

```diff
@@ -27,12 +27,15 @@ export const StyledTableContainer = styled(TableContainer, {
 }));
 
 export const StyledTable = styled(Table, {
-  shouldForwardProp: (prop: string) => prop !== 'customWidth',
-})(({ customWidth }) =>
+  shouldForwardProp: (prop: string) => prop !== 'customWidth' && prop !== 'isBorderRight' && prop !== 'theme',
+})(({ customWidth, isBorderRight, theme }) =>
   customWidth
     ? {
         tableLayout: 'fixed',
         width: 'min-content',
+        borderWidth: isBorderRight && '0px 1px 0px 0px',
+        borderStyle: isBorderRight && 'solid',
+        borderColor: theme.background.isDark ? COLORING.DARK_MODE_BORDER : COLORING.BORDER_LIGHT,
```

Comment:
we already do this calculation, the information is on the styling object

Reply:
it `styling.body.borderRightColor`?